managed/src/main/java/com/yugabyte/yw/common/CertificateHelper.java

```diff
@@ -151,109 +154,97 @@ public static UUID createRootCA(String nodePrefix, UUID customerUUID, String sto
   }
 
   public static JsonNode createClientCertificate(UUID rootCA, String storagePath, String username,
```

Comment:
If all these are ultimatly going to result in YWSE(INTERNAL_ERROR) then just catch these exception s in this method and throw throw YWSE(INTERNAL_ERROR)
This avoids poluting method signitures with checked exceptions (Assuming that all of these are getting  treated in a same manner - as 500)

Reply:
Yeah, make sense.